packages/web/src/dev/ValidatorsListRow.tsx

```diff
@@ -91,19 +100,46 @@ class ValidatorsListRow extends React.PureComponent<Props & I18nProps, State> {
     this.removeDocumentListener()
   }
 
+  isPinned(toggle?: boolean) {
+    const { address } = this.props.group
+    let list = (localStorage.getItem(localStoragePinnedKey) || '').split(',') || []
+    let isPinned = list.includes(address)
+    if (toggle) {
+      list = list.filter((_) => _ !== address)
+      if (!isPinned) {
+        list.push(address)
+      }
+      isPinned = !isPinned
+      localStorage.setItem(localStoragePinnedKey, list.join(','))
+      this.props.onPinned()
+    }
+    return isPinned
+  }
+
   render() {
     const { group, expanded } = this.props
-    const { tooltip } = this.state
+    const { tooltip, isPinned } = this.state
     const stopPropagation = (event: any) => {
       event.stopPropagation()
     }
     const toggleTooltip = (event: any) => {
       stopPropagation(event)
       this.setState({ tooltip: !tooltip })
     }
+    const togglePinned: any = (event: any) => {
```

Comment:
nit: remove `any`

Reply:
It produces an error because technically the view doesn't emit the event (but it does). Another solution is removing the `any` and start using a `div`